### PR TITLE
#1/Get-a-List-of-Meals

### PR DESCRIPTION
**Pull request by omorgenroth**
Mehr Details im [Issue](https://github.com/omorgenroth/capstone-project/issues/1)

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -1,43 +1,21 @@
...
-    <!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
+<html lang="de">
```
Comment (FabianAlleblas) [inline frontend/public/index.html:2]: Nice 👍

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,25 +1,32 @@
...
+        <Route path="/dishes">
+          <DishOverviewPage dishes={dishes} />
+        </Route>
+      </Switch>
     </div>
-  );
+  )
+
+  function addIsCheckedValue(data) {
+    return data.map((o) => ({ ...o, isChecked: true }))
```
Comment (FabianAlleblas) [inline frontend/src/App.js:28]: What is o?

File: frontend/src/Components/DishItem.test.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { render } from '@testing-library/react'
+import DishItem from './DishItem'
+
+describe('DishItem', () => {
+  it('renders with the CheckedIcon if isChecked is true', () => {
+    const { getByTestId } = render(<DishItem isChecked={true} name="My Dish" />)
+
+    expect(getByTestId('checked')).toBeInTheDocument()
+  })
+})
```
Comment (FabianAlleblas) [inline frontend/src/Components/DishItem.test.js:10]: 💪

File: frontend/src/App.js (hunk 2 above)
Comment (omorgenroth) [reply frontend/src/App.js]: o was shortform for "object" in my head. But changed it now for the sake of clarity.

**Pull request merged**